src/beautifiers/beautysh.coffee

```diff
@@ -0,0 +1,15 @@
+"use strict"
+Beautifier = require('./beautifier')
+
+module.exports = class BashBeautify extends Beautifier
+  name: "beautysh"
+  link: "https://github.com/bemeurer/beautysh"
+  options: {
+    Bash:
+      indent_size: true
```

Comment:
Actually, I regenerated the docs (ran both `codo` and `npm docs`) but to no avail. 
I thought maybe generic parameters didn't show but apparently they should.
Can't find what I'm missing, do you have any idea ?